package.json

```diff
@@ -44,6 +44,7 @@
     "rimraf": "^3.0.2",
     "secret-stack": "6.3.1",
     "ssb-caps": "1.1.0",
+    "ssb-config": "^3.4.5",
```

Comment:
this doesn't look like its used?

Reply:
Oh, sorry, I used that for a non-committed example.js where I'm testing on my production ssb. I can remove it if you wish

Reply:
Yeah lets remove that :)